template-for-repository/proofs/Makefile.common

```diff
@@ -138,6 +138,22 @@ ifneq ($(strip $(EXTERNAL_SAT_SOLVER)),)
 endif
 CHECKFLAGS += $(USE_EXTERNAL_SAT_SOLVER)
 
+# Job pools
+# If Litani is new enough (`litani print-capabilities` prints "pools"), proofs
```

Comment:
Clarified the text here. It will use this feature only if the Litani version is new enough. You can figure out if the version is new enough by running `litani print-capabilities` and seeing if the output includes `"pools"`.